<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InGameScene</class>
 <widget class="QDialog" name="InGameScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>414</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_notcall">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>NotCall</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_1p">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>1point</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_2p">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>2points</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_3p">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>3points</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_play">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_hint">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>hint</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_pass">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>pass</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_ai1Remain">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AI1 left: 17</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ai2Remain">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AI2 left: 17</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
